sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensor.java

```diff
@@ -41,17 +41,28 @@
   private final CoverageConfiguration coverageConf;
   private final CoverageAggregator coverageAggregator;
   private final boolean isIntegrationTest;
+  private final String languageKey;
+  private final String languageName;
 
-  public CoverageReportImportSensor(CoverageConfiguration coverageConf, CoverageAggregator coverageAggregator, boolean isIntegrationTest) {
+  public CoverageReportImportSensor(CoverageConfiguration coverageConf, CoverageAggregator coverageAggregator, String languageKey, String languageName, boolean isIntegrationTest) {
     this.coverageConf = coverageConf;
     this.coverageAggregator = coverageAggregator;
     this.isIntegrationTest = isIntegrationTest;
+    this.languageKey = languageKey;
+    this.languageName = languageName;
   }
 
   @Override
   public void describe(SensorDescriptor descriptor) {
-    descriptor
-      .name("Coverage Report Import");
+    StringBuilder builder = new StringBuilder(this.languageName);
+    if (this.isIntegrationTest) {
+      builder.append(" Integration Tests ");
+    } else {
+      builder.append(" Unit Tests ");
+    }
+    builder.append("Coverage Report Import");
+    descriptor.name(builder.toString());
```

Comment:
Do we need a StringBuilder for concatenating two strings? Why not:

    if (isIntegrationTest) {
        descriptor.Name(" Integration Tests Coverage Report Import");
    } else {
        descriptor.Name(" Unit Tests Coverage Report Import");
    }